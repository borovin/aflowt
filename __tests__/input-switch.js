import test from 'ava'
import Browser from './helpers/browser'
import {prettyPrint} from 'html'

test.beforeEach(t => {
  t.context.browser = new Browser()

  return t.context.browser.goto('/examples/input-switch.html')
})

test.afterEach(t => {
  return t.context.browser.close()
})

test('Checked switch', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-switch[name="switch_1"]')

  t.snapshot(prettyPrint(snapshot))
})

test('Unchecked switch', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-switch[name="switch_2"]')

  t.snapshot(prettyPrint(snapshot))
})
