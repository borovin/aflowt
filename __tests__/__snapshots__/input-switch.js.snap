// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked switch 1`] = `
"<b-input-switch name=\\"switch_1\\" checked=\\"\\">
    <label>
        <input name=\\"switch_1\\" type=\\"checkbox\\" checked=\\"\\">
        <b-input-switch--icon></b-input-switch--icon>
        <slot></slot>
    </label>
</b-input-switch>"
`;

exports[`Unchecked switch 1`] = `
"<b-input-switch name=\\"switch_2\\">
    <label>
        <input name=\\"switch_2\\" type=\\"checkbox\\">
        <b-input-switch--icon></b-input-switch--icon>
        <slot>switch button</slot>
    </label>
</b-input-switch>"
`;
